### PR TITLE
Update kubeflow/katib manifests from v0.15.0-rc.0

### PR DESCRIPTION
**Pull request by DomFleischmann**
Update Katib manifests to v0.15.0-rc0.

@johnugeorge @andreyvelich @kimwnasptd

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ This repo periodically syncs all official Kubeflow components from their respect
 | Jupyter Web App | apps/jupyter/jupyter-web-app/upstream | [v1.6.0-rc.1](https://github.com/kubeflow/kubeflow/tree/v1.6.0-rc.1/components/crud-web-apps/jupyter/manifests) |
 | Tensorboards Web App | apps/tensorboard/tensorboards-web-app/upstream | [v1.6.0-rc.1](https://github.com/kubeflow/kubeflow/tree/v1.6.0-rc.1/components/crud-web-apps/tensorboards/manifests) |
 | Volumes Web App | apps/volumes-web-app/upstream | [v1.6.0-rc.1](https://github.com/kubeflow/kubeflow/tree/v1.6.0-rc.1/components/crud-web-apps/volumes/manifests) |
-| Katib | apps/katib/upstream | [v0.14.0-rc.0](https://github.com/kubeflow/katib/tree/v0.14.0-rc.0/manifests/v1beta1) |
```
Comment (johnugeorge) [inline README.md:53]: Didn't we update the images in the master after the v0.14.0 release ?  When do we branch out in manifests repo?

**Comment by kimwnasptd** [comment]
Looks like the GH Action is still failing in the KinD cluster with Katib's mysql Pod, like in the previous release https://github.com/kubeflow/manifests/pull/2273

Will take a look. But most probably we might merge this PR even though the action is failing for now

**Comment by DomFleischmann** [comment]
@kimwnasptd if you believe that this has to do with the GH Action I'm okay with merging this

**Comment by kimwnasptd** [comment]
@DomFleischmann could you rebase on top of the latest `master` to resolve the conflicts?

I'll also run the test we have locally. I remember that this would succeed in my laptop, and we had the hunch that this had to do with the GH Action VM. If it works locally I'll merge this manually and look at the test orthogonally

**Comment by kimwnasptd** [comment]
I manually run the test in my laptop and the Experiment completed as expected. I'll manually merge the PR and look into the test in parallel

/lgtm
/approve

**Pull request merged**